clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/cache/model/Application.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Netflix, Inc.
```

Comment:
license attributions are all over the place, pls fix :P

Reply:
fixed!